src/extrovermectin.ts

```diff
@@ -208,19 +220,46 @@ function initializeDireWarren(): void {
   if (options.some((option) => banishedMonsters.get(option) === $monster`fluffy bunny`)) {
     return;
   }
-  if (banishedMonsters.get($skill`Furious Wallop`) === $monster`fluffy bunny`) return;
+  if (banishedMonsters.get($skill`Batter Up!`) === $monster`fluffy bunny`) return;
```

Comment:
q: what did the line cook at waffle house say to his buddy when he finished mixing the waffle stuff

a: "batter up!"

i am personally illegal in at least 26 states